quick image link fix

thesuperRL opened the pull request:
I forgot one discord link, this updates it

euphoric3hub commented:
This is the same as previous pr and i feel this is more feasible , so I would like to continue the same

linkai101 reviewed (changes requested):
I'm a bit confused since this looks exactly like the merged proposal by JackTDC. (https://github.com/hackclub/jams/pull/13)

Also, looks like you created a new folder called "jam" and deleted the example file -- please keep example.md and add your proposal file in the "jams" folder instead.

euphoric3hub commented:
> I'm a bit confused since this looks word for word exactly like the merged proposal by JackTDC. (#13)
> 
> Also, looks like you created a new folder called "jam" and deleted the example file -- please keep example.md and add your proposal file in the "jams" folder instead.

hey JackTDC is not working on this jam, so Thomas asked me to take up on this project as I previously proposed the same jam unknowingly and later deleted it

linkai101 commented:
> > I'm a bit confused since this looks word for word exactly like the merged proposal by JackTDC. (#13)
> > Also, looks like you created a new folder called "jam" and deleted the example file -- please keep example.md and add your proposal file in the "jams" folder instead.
> 
> hey JackTDC is not working on this jam, so Thomas asked me to take up on this project as I previously proposed the same jam unknowingly and later deleted it

Ah if that's the case then no problem. 👍

thesuperRL committed:
quick image link fix

thesuperRL committed:
change the back button so that it works

in batches, clicking "back" formerly just went back to index

this fixes it so that it goes back to the batches page

at the cost of making the browser back button not remember correctly (since router.back) is equivalent to pressing browser back button

thesuperRL commented:
oops I put the two of them together

thesuperRL commented:
commit wrong
